src/react/account/details/__tests__/Locations.test.jsx

```diff
@@ -150,7 +150,7 @@ describe('Locations', () => {
         const thirdRowColumns = thirdRow.find(T.Cell).map(column => column.text());
         expect(thirdRowColumns.length).toEqual(nbrOfColumnsExpected);
         expect(thirdRowColumns[0]).toEqual('location-file');
-        expect(thirdRowColumns[1]).toEqual('Zenko Local Filesystem');
```

Comment:
Sorry to be the "bad cop" here but it seems to be some other places (especially in location creation) where "zenko" is mentioned. Can you please look at it @theocerutti ? Also, we do not need to change it if Zenko appears in a doc file or comments, only for the rendered code. Thank you.

Reply:
done